pkg/daemon/daemon.go

```diff
@@ -200,6 +210,41 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 	return nil
 }
 
+func populatePtp4lConf(tomlContent *string, output *ptp4lconf) {
```

Comment:
I will update the commit message.  I tried BurntSushi toml parser, but it doesn't work as the config file isn't valid toml (because of using spaces instead of =).  I would be more worried of unknowingly breaking something if I try switching " " to "=" to parse, and then switch back after.